src/components/MaterialUI/ScopeSelector/ScopeSelector.test.js

```diff
@@ -91,11 +91,12 @@ describe("ScopeSelected", () => {
 										<InputBase inputProps={inputBaseProps} />
 									</div>
 									<div>
-										<TreeView
+										<ScopeTreeView
 											rootId="Global"
 											getScope={getScope}
-											selectedScope={selectedScope}
-											closeSelector={closeSelector}
+											selected={selectedScope.id}
+											expanded={selectedScope.scopePath}
+											onSelected={closeSelector}
 										/>
```

Comment:
Is it possible for the scope too be empty?

Reply:
no